frontend/src/citizen-frontend/decisions/decision-response-page/DecisionResponse.tsx

```diff
@@ -78,33 +79,42 @@ export default React.memo(function DecisionResponse({
   }
 
   const onSubmit = async () => {
+    if (parsedDate === null) throw new Error('Parsed date was null')
     setSubmitting(true)
     return (acceptChecked
-      ? acceptDecision(applicationId, decisionId, requestedStartDate)
+      ? acceptDecision(applicationId, decisionId, parsedDate)
       : rejectDecision(applicationId, decisionId)
```

Comment:
actually I think there's a bug now, sorry missed it when we pair-coded. I guess the null-check on parsedDate should be done only when accepting, since rejection does not care about the date